executor/aggfuncs/func_group_concat.go

```diff
@@ -72,7 +72,7 @@ func (e *baseGroupConcat4String) AppendFinalResult2Chunk(_ sessionctx.Context, p
 
 func (e *baseGroupConcat4String) handleTruncateError(sctx sessionctx.Context) (err error) {
 	if atomic.CompareAndSwapInt32(e.truncated, 0, 1) {
-		if !sctx.GetSessionVars().StmtCtx.TruncateAsWarning {
+		if !sctx.GetSessionVars().StmtCtx.TypeConvContext.Flags().TruncateAsWarning() {
```

Comment:
Do you think rename `TypeConvContext` to `TypeContext` is better? I realized the previous is too verbose

Reply:
YES!!!!!